lm_eval/api/task.py

```diff
@@ -295,6 +297,16 @@ def _process_doc(self, doc):
             The processed version of the specified `doc`.
         """
         return doc
+    
+    def create_choices(self, doc):
+        if self._config.create_choices is None:
+            return ast.literal_eval(
+                    utils.apply_template(
+                        self._config.template_aliases + "{{answer_choices}}", doc
+                        )
+                    )
+        else:
```

Comment:
if _config.create_choices is a string, it won’t be handled here—I think we need a 3rd case that does utils.apply_template(self._config.create_choices, doc) if it’s of type string